TASVideos/Pages/Shared/Components/RejectedSubmissions/Default.cshtml

```diff
@@ -1,4 +1,7 @@
 ﻿@model IEnumerable<RejectedSubmission>
```

Comment:
Change this to ICollection, and use the .Count property instead of method.

Since you are enumerating multiple times (foreach and Count()), it is better to use a contract that ensures this is safe